docs/book/how-to/infrastructure-deployment/stack-deployment/export-stack-requirements.md

```diff
@@ -0,0 +1,13 @@
+---
+description: Export stack requirements
+---
+
+You can get the `pip` requirements of your stack by running the `zenml stack export-requirements <STACK-NAME>` CLI command.
```

Comment:
This chapter IMO is not about exporting stack requirements. It's about telling a story about:

- Why do we need to install stack requirements locally and in the execution environment?
- What does it mean to export requirements? (i.e. that these requirements are the suggested ranges of values that version of zenml is released with)
- A reason why someone would do this in a production environment
- What is the link between a requirements file generated by this file vs the stack integration installing dependencies that zenml already does when it builds a docker image